app/helpers/deployment-helpers.ts

```diff
@@ -30,7 +30,7 @@ const MAX_DOMAIN_LENGTH = 63;
 const BRANCH_HASH_LENGTH = 8;
 
 export function getBranchHandle(branchName: string) {
-  const handle = branchName.replace(/\//g, "-").replace(/_/g, "-");
+  const handle = branchName.replace(/[\/_,]/g, "-");
```

Comment:
Maybe we should instead *reverse* this. i.e. `replace(/[^a-zA-Z0-9-]+/, "-")` Because branch names can contain a lot of things not in this list.

Reply:
Agreed, we might as well account for the other cases now.

Reply:
Addressed in [2f79960](https://github.com/iFixit/govinor/pull/6/commits/2f79960b8bbf6a20d9a9f7af9c1ee5661809c55d)